eng/feeds/azure-scaffolding.json

```diff
@@ -47,7 +47,7 @@
     ],
     "inputs": {
       "ServiceNamespace": {
-        "description": "Please provide service namespace in Pascal case:",
+        "description": "Please provide service namespace in Pascal case(e.g. Azure.Contoso):",
```

Comment:
I wonder why we still need the service namespace.

Reply:
This one is for data plane where the full namespace is expected for arm template just the rp name is expected without Microsoft.

Reply:
I hope we are not using this as the service name? `specification/contoso/data-plane/Azure.Contoso` or `specification/Azure.Contoso` are not recommended in folder structure v2.

Reply:
tsp init has to be run in the folder already created so this is not used